secrets-provider/templates/secrets-provider.yaml

```diff
@@ -48,15 +65,12 @@ spec:
         - name: CONJUR_AUTHN_LOGIN
           value:  {{ .Values.environment.conjur.authnLogin | quote }}
 
-        - name: SUPPORT_MULTIPLE_APPS
```

Comment:
removing in order to avoid introducing env variables without the code. Will bring this back in when we make this addition in the code

Reply:
good idea